File: src/theme/colors.ts
// "Quiet luxury" palette from the Claude Design export (An copy.dc.html).
// Sky-blue backgrounds so Mây stands over them; jade / rose / lavender accents.
// Dark backgrounds are reserved for meditation sessions and Premium screens.

export const colors = {
  ink: '#1B2420',
  inkSoft: 'rgba(27,36,32,0.7)',
  inkMuted: 'rgba(27,36,32,0.55)',
  inkFaint: 'rgba(27,36,32,0.45)',
  inkHairline: 'rgba(27,36,32,0.08)',

  sage: '#6F9E8A',
  sageDeep: '#4E7A68',
  sageTint: '#E3EDE7',
  sageTintText: '#3D6555',

  lavender: '#A79BC4',
  lavenderTint: '#EDE9F4',
  lavenderTintText: '#4F4468',

  rose: '#F6BDD4',
  blueGray: '#8496B8',
  tan: '#C4A38B',

  premiumMint: '#CFE2DB',
  premiumDarkA: '#20423C',
  premiumDarkB: '#16201E',
  premiumDarkC: '#101817',

  danger: '#8C4A45',

  appBg: '#EAF0F4',
  card: '#FFFFFF',
  cardMuted: 'rgba(255,255,255,0.7)',

  skyTop: '#D7E9F4',
  skyMid: '#B7D4E6',
  skyBottom: '#98BFD6',

  loginTop: '#DCECF5',
  loginMid: '#C3DCEA',
  loginBottom: '#A6CAD9',

  sessionDarkA: '#22423A',
  sessionDarkB: '#16201E',
  sessionDarkC: '#0E1615',

  white: '#FFFFFF',
  onDark: '#FFFFFF',
  onDarkSoft: 'rgba(255,255,255,0.82)',
  onDarkMuted: 'rgba(255,255,255,0.55)',
  onDarkFaint: 'rgba(255,255,255,0.42)',
  onDarkHairline: 'rgba(255,255,255,0.16)',
} as const;

export const moodColors: Record<string, string> = {
  'binh-yen': colors.sage,
  vui: colors.rose,
  'binh-thuong': 'rgba(27,36,32,0.25)',
  'lo-lang': colors.lavender,
  buon: colors.blueGray,
  'kiet-suc': colors.tan,
};

export const moodLabels: Record<string, string> = {
  'binh-yen': 'Bình yên',
  vui: 'Vui',
  'binh-thuong': 'Bình thường',
  'lo-lang': 'Lo lắng',
  buon: 'Buồn',
  'kiet-suc': 'Kiệt sức',
};

export type MoodKey = keyof typeof moodLabels;

File: lib/theme/colors.dart
import 'package:flutter/material.dart';
import '../models/mood.dart';

/// "Quiet luxury" palette from the Claude Design export (An copy.dc.html).
/// Sky-blue backgrounds so Mây stands over them; jade / rose / lavender
/// accents. Dark backgrounds are reserved for meditation sessions and
/// Premium screens.
class AppColors {
  AppColors._();

  static const ink = Color(0xFF1B2420);
  static Color inkSoft = ink.withValues(alpha: 0.7);
  static Color inkMuted = ink.withValues(alpha: 0.55);
  static Color inkFaint = ink.withValues(alpha: 0.45);
  static Color inkHairline = ink.withValues(alpha: 0.08);

  static const sage = Color(0xFF6F9E8A);
  static const sageDeep = Color(0xFF4E7A68);
  static const sageTint = Color(0xFFE3EDE7);
  static const sageTintText = Color(0xFF3D6555);

  static const lavender = Color(0xFFA79BC4);
  static const lavenderTint = Color(0xFFEDE9F4);
  static const lavenderTintText = Color(0xFF4F4468);

  static const rose = Color(0xFFF6BDD4);
  static const blueGray = Color(0xFF8496B8);
  static const tan = Color(0xFFC4A38B);

  static const premiumMint = Color(0xFFCFE2DB);
  static const premiumDarkA = Color(0xFF20423C);
  static const premiumDarkB = Color(0xFF16201E);
  static const premiumDarkC = Color(0xFF101817);

  static const danger = Color(0xFF8C4A45);

  static const appBg = Color(0xFFEAF0F4);
  static const card = Color(0xFFFFFFFF);

  static const skyTop = Color(0xFFD7E9F4);
  static const skyMid = Color(0xFFB7D4E6);
  static const skyBottom = Color(0xFF98BFD6);

  static const loginTop = Color(0xFFDCECF5);
  static const loginMid = Color(0xFFC3DCEA);
  static const loginBottom = Color(0xFFA6CAD9);

  static const sessionDarkA = Color(0xFF22423A);
  static const sessionDarkB = Color(0xFF16201E);
  static const sessionDarkC = Color(0xFF0E1615);
}

const Map<Mood, Color> moodColors = {
  Mood.binhYen: AppColors.sage,
  Mood.vui: AppColors.rose,
  Mood.binhThuong: Color(0x401B2420),
  Mood.loLang: AppColors.lavender,
  Mood.buon: AppColors.blueGray,
  Mood.kietSuc: AppColors.tan,
};

const Map<Mood, String> moodLabels = {
  Mood.binhYen: 'Bình yên',
  Mood.vui: 'Vui',
  Mood.binhThuong: 'Bình thường',
  Mood.loLang: 'Lo lắng',
  Mood.buon: 'Buồn',
  Mood.kietSuc: 'Kiệt sức',
};
